lib/src/nuvigator.dart

```diff
@@ -346,7 +351,7 @@ class NuvigatorState<T extends Router> extends NavigatorState
   Widget build(BuildContext context) {
     Widget child = super.build(context);
     if (widget.wrapper != null) {
-      child = widget.wrapper(context, child);
+      child = widget.wrapper(context, child, stateTracker.currentRouteName);
```

Comment:
Passing the routeName here does not make much sense given this wrapper is applied around the Navigator, and not on a specific screen